python/cudf/cudf/core/dataframe.py

```diff
@@ -2642,7 +2642,15 @@ def query(self, expr, local_dict={}):
         return result
 
     @applyutils.doc_apply()
-    def apply_rows(self, func, incols, outcols, kwargs, cache_key=None):
+    def apply_rows(
+        self,
+        func,
+        incols,
+        outcols,
+        kwargs,
+        pessimistic_nulls=True,
```

Comment:
Can you add this to the docstring as to what this does?

Reply:
done.